### PR TITLE
Makes prying up conveyors take time and visible to others

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
tweak: Conveyors have been more firmly anchored. No fun allowed
/:cl:

Buffs auto-borger out of the trash heap. its just long enough, without being too generous.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
now my emagged disposals grinder traps are truly inescapable

**Comment by Supermichael777** [comment]
the real trick is prying up the first conveyor for a banana peel. or moving the grinder one tile away so they cant get away.

**Comment by ExcessiveUseOfCobblestone** [comment]
Welcome to Hell

!img[](http://i2.kym-cdn.com/entries/icons/original/000/011/512/Mrbones.png)

**Comment by ExcessiveUseOfCobblestone** [comment]
MrBones

Hello!

**Comment by RandomMarine** [comment]
"Reports of the automated processing of humans and human remains have been determined to be misinformation. Human remains are only ever handed with dignity and respect on NT stations."

What the fuck is this even supposed to mean to the average player? How are they supposed to figure out this refers to autoborgers? These vague IC changelog entries are horrible.

**Comment by Cyberboss** [comment]
What's an auto-borger?

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -116,12 +116,15 @@
...
 	if(istype(I, /obj/item/weapon/crowbar))
-		if(!(stat & BROKEN))
-			var/obj/item/conveyor_construct/C = new/obj/item/conveyor_construct(src.loc)
-			C.id = id
-			transfer_fingerprints_to(C)
-		user << "<span class='notice'>You remove the conveyor belt.</span>"
-		qdel(src)
+		user.visible_message("<span class='notice'>[user] struggles to pry up \the [src] with \the [I].</span>", \
+		"<span class='notice'>You struggle to pry up \the [src] with \the [I].</span>")
+		if(do_after(user, 40/I.toolspeed, target = src))
```
Comment (optimumtact) [inline code/modules/recycling/conveyor2.dm]: check !qdeleted after teh do after finishes to prevent people deconstructing it multiple times

Comment (optimumtact) [inline code/modules/recycling/conveyor2.dm]: @Supermichael777

**Pull request merged**